app/components/utils/generic/timeline/timeline.hbs

```diff
@@ -0,0 +1,9 @@
+<div>
```

Comment:
Een tijdslijn met acties komt voor onder /vertalingen /drukproeven & /publicaties. Is deze component zinvol?

Reply:
Ik wist niet goed of ik die zou als "is nodig" moest interpreteren. Voorlopig heb ik het laten zitten. Ik kan daar beter aan uit.